ui-tests/src/test/java/org/openmrs/reference/page/RegistrationSummaryPage.java

```diff
@@ -0,0 +1,25 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class RegistrationSummaryPage extends Page {
+    private static final By RELATIONSHIP = By.id("relationships-info-edit-link");
+    private static final By DEMOGRAPHICS = By.id("demographics-edit-link");
+    private  static final By CONTACT_INFO = By.id("contactInfo-edit-link");
+
+  
```

Comment:
Remove one of the redundant line code `LN-10` or `LN-11` for quality assurance.

Reply:
Resolved thanks